.github/workflows/pg-9-migtests.yml

```diff
@@ -85,7 +85,7 @@ jobs:
           docker run -d --name yugabytedb \
             -p7000:7000 -p9000:9000 -p15433:15433 -p5433:5433 -p9042:9042 \
             yugabytedb/yugabyte:${{ matrix.version }} \
-            bin/yugabyted start --background=false --ui=false
+            bin/yugabyted start --tserver_flags="ysql_hba_conf_csv={host all yugabyte all trust,host all all all md5}" --background=false --ui=false 
```

Comment:
Shall we just have `host all all all md5` ? Just want to avoid "trust" altogether 😅

Reply:
yeah I started with that only, but somehow yugabyted wasn't coming up with that conf as yugabyte is a default user maybe trust is required for that, I can take that up later after figuring out how to do that